packages/server/README.md

```diff
@@ -140,11 +140,12 @@ finished uploads. (`boolean`)
 #### `options.onUploadCreate`
 
 `onUploadCreate` will be invoked before a new upload is created.
-(`(req, res, upload) => Promise<res>`).
+(`(req, res, upload) => Promise<{ res: http.ServerResponse, metadata?: Record<string, string>}>`).
 
-If the function returns the (modified) response, the upload will be created. You can
-`throw` an Object and the HTTP request will be aborted with the provided `body` and
-`status_code` (or their fallbacks).
+- If the function returns the (modified) response the upload will be created.
```

Comment:
Does this mean that this change is backwards compatible? I.e. `onUploadCreate` implementations from before will continue working?

Reply:
Correct, hence the ugly temporary code/types. Both versions are also tested (another existing test not visible in the diff)